.github/workflows/golangci-lint.yml

```diff
@@ -4,6 +4,7 @@ on:
   #   tags:
   #     - v*
   #   branches:
+  #     - master
```

Comment:
Just a head up, this is not actually doing anything, since it's commented out.

Reply:
Yeah, that's just a consequence of re-running init/copy-paste over top of the existing files. The general premise is trying to keep it so that the future re-inits minimize changes.￼

Who knows if that will actually work out. ￼🤷‍♂️ (This strategy made a ton of difference when I was using Ember CLI generators so it became a habit as a way to minimize divergence.)